.github/workflows/go.yml

```diff
@@ -1,13 +1,40 @@
 name: Go
 on: [push]
 jobs:
+    build:
+        name: Build
+        runs-on: ubuntu-latest
+        steps:
 
-  build:
-    name: GoCity CI
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@master
-      - uses: actions/setup-go@v1
-        with:
-          version: '1.12'
-      - run: go test -race -cover ./...
+            - name: Set up Go 1.12
+              uses: actions/setup-go@v1
+              with:
+                  go-version: 1.12.4
+
+            - name: Check out code into the Go module directory
+              uses: actions/checkout@v1
+              with:
+                  fetch-depth: 1
+                  path: go/src/github.com/user/gocity
+
+            - name: GoLint
+              run: |
+                  go get golang.org/x/lint/golint
+                  /home/runner/work/gocity/go/bin/golint ./
+              env:
+                  GOPATH: /home/runner/work/gocity/go
+            - name: Go get dependecies
+              run: |
+                  go get -u github.com/rodrigo-brito/gocity/handle
+                  go get -u github.com/stretchr/testify/assert
+                  go get -u github.com/rodrigo-brito/gocity/lib
+                  go get -u github.com/rodrigo-brito/gocity/utils
```

Comment:
The `actions/checkout@v1` steps downloads all the project, right? Why you need download again? With go get, it will download an old version of the project, maybe it can be a problem.
To download the dependencies, you can use `go mod download`.

Reply:
I removed the dependency download part after making sure everything else works fine, in the new commit.